styles.css

```diff
@@ -36,18 +36,28 @@ h2 {
   border-radius: 2.5px;
 }
 
+/* container for the users drop down menu */
```

Comment:
Don't add comments like this, just rely on good, clear identifiers. Your names should describe the code just fine.